src/webportal/src/app/job/job-view/fabric/job-detail/conn.js

```diff
@@ -134,6 +134,11 @@ export async function cloneJob(rawJobConfig) {
   const plugins = window.PAI_PLUGINS;
   const pluginIndex = plugins.findIndex((x) => x.id === pluginId);
   if (pluginIndex === -1) {
+    // redirect v2 job to default submission page
+    if (isJobV2(rawJobConfig)) {
+      window.location.href = `/submit.html?${qs.stringify(query)}`;
```

Comment:
I think we should still pop up a warning message to user.

Reply:
sure, will fix it